### PR TITLE
Add Documentation Working Group Proposal

### PR DESCRIPTION
**Pull request by HarshCasper**
## Background for the Proposal

Back in the TSC Meeting for the month of April, @aornugent proposed the setup of working groups to better organize our efforts. This document proposes the setup of a **Documentation Working Group** which will overall work forward for the benefit and growth of the Open-Source documentation hosted at moja global. 

## How does it help? 

Working Groups will hopefully be used for capability building. Each Working Group will require one or more Coordinators. Working Groups might also request feedback and support from the moja global Management, Strategy and Advisory boards. The working groups are all about communication rather than accountability. 

Accountability is instead delegated to the individuals responsible for a specific project who choose to report their progress to the Working Group for oversight and collaboration. Such reports allow Working Groups to maintain an overview of ongoing development, published on a moja global website or repository to help the community track who is working on what.

## Final Notes 

The proposal uncovers most of the Terms of References around what would the Documentation Working Group do and how will it contact the overall TSC and the impact that it would bring. 

The rest of the questions that would be uncovered as we onboard contributors would be: 

- What are explicitly non-goals of your group?
- How would you like to conduct evaluations of working group progress?
- Who are the initial Coordinators? How long is their term?
- Where do you see your group needing help?

As of now the proposal is open for review and feedback from the community and I would be grateful to have people pitch in for leading the documentation efforts at moja global.

### CODE REVIEW

**Comment by sarahhaggarty** [comment]
Nice proposal @HarshCasper!

I wanted to check what you mean with "For every project release"? Does it mean that there is always one project delivered per release? Or could multiple projects be delivered in a release?

And can you give an example of "What are explicitly non-goals of your group?" Are these like stretch goals? So they are nice to have but not mandatory for the project work to be considered completed?

Thanks for the hard work!

**Comment by aornugent** [comment]
Hey @HarshCasper - great to see this progress, thank you for putting together your working group proposal.

I've read through and I think it's great. I have one substantial suggestion that I'd like to make before submitting a detailed review. I think you can treat the PR itself as 'proposing the idea' and instead change the tense of your document to be present tense. This means instead of talking about what the documentation working group 'will do' then the talk about what the group does.

This way, the document we finally publish won't be a proposal to create a group but instead be the terms of reference that the group follow to define their actions. 

@sarahhaggarty - I take 'non-goals' to mean things the working group are not pursuing, but I get your interpretation too. We should make it clearer in the suggested TOR criteria that are listed in the document here: https://github.com/moja-global/Google_Season_of_Documentation/issues/12

**Comment by HarshCasper** [comment]
Hi @aornugent @sarahhaggarty 

Can you please have a review over the same, again?

File: Documentation-Working-Group-Proposal.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+1.  Research: Documentation research process is an intensive one and we depend on the developer inputs while performing the research.
+
+2.  Draft: Based on the prior research done, a draft can be prepared which is then reviewed by the stakeholders before signing it off.
+
+3.  Review: After the release of the final draft, the developers/maintainers review the draft and suggest improvements and provide feedback before it is sent as a Pull Request.
+
+4.  Merge: The documentation is peer reviewed, and after all suggestions are incorporated, the write-up is merged to the official documentation and hosted.
+
+But where do we plan for documentation releases and developing new documentation pieces? The Documentation issues are created on the respective repositories through GitHub Issues, where they are tracked and accounted for by the stakeholders. All the documentation-related issues would be labelled as “documentation”. At the end of the day, as a volunteer for the Documentation Working group we would be working on:
```
Comment (aornugent) [inline Documentation-Working-Group-Proposal.md:27]: I suggest you reformat the question here to be a statement and make it a subheading. You might like to think of subheadings for each paragraph for this section.

**Comment by aornugent** [comment]
This looks really good Harsh, happy for you to merge. Let's try to get it up on the contributors site ASAP.

File: Documentation-Working-Group-Proposal.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+-   Creating user-story based end user documentation that makes complex things simple so that even a first time user can quickly start off using our projects.
+
+-   Creating modular content that can be reused/repurposed for different formats and different use cases.
+
+-   Adhering to minimalism practices, logically structure our content, and endeavor to make it as findable as possible.
+
+-   Reduce the number of developer/user issues the engineering team needs to deal with on GitHub.
+
+As part of the Documentation Working group, we work asynchronously with the engineering teams to set a clear Release, feature freeze, and code Freeze dates, which allows us to set our Documentation Freeze and even Release Notes dates.
```
Comment (sarahhaggarty) [inline Documentation-Working-Group-Proposal.md:37]: freeze or Freeze? Would be good to make the capitalisation uniform so the reader can understand if this is a moja global term or not.

**Review by sarahhaggarty — COMMENTED** [review]
This is looking great Harsh! I made a few very minor suggestions for you to consider. It would also be cool to standardise on either "documentation" or "Documentation" throughout the document.

Keep up the great work!

File: Documentation-Working-Group-Proposal.md (hunk 2 above)
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:37]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Setting up the Documentation Working Group
+
+## Introduction
+
+moja global is an international collaboration under the Linux Foundation that provides software tools to better manage the land sector (forests and agriculture). The flagship software is called the **Full Lands Integration Tool** (FLINT) and it is used to estimate emissions and removals of greenhouse gases from the land sector. The FLINT uses a range of input data and combines it with model greenhouse gas fluxes between a range of pools for every time step. At every time step, all pool values and the fluxes between all pools are stored in an output database. Reports are generated by a reporting tool that queries the database and arranges the results in the required format.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:5]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## How to apply?
+
+The Documentation Working Group is entirely driven by volunteer efforts but we like to keep track of all the contributors who would like to join. It help us set proper workflows, processes and contribution pipelines that allows us to track the work being done and set realistic timelines. You can pitch for several documentation-specific domains that you would like to contribute to.
+
+All the Contributors for the Documentation Working Group are roped in for a bi-weekly sync call where we discuss the work that has been completed and the upcoming documentation work that needs to be done.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:59]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+We are yet to decide on the tools & technologies that we would be employing for our purpose. However some of them that are most commonly employed are [Markdown](https://guides.github.com/features/mastering-markdown/), [Docusaurus](https://docusaurus.io/), [Sphinx](https://www.sphinx-doc.org/), [ReadTheDocs](https://readthedocs.org/), [Moodle](https://moodle.org/), [GitHub Issues](https://guides.github.com/features/issues/), [GitHub Project Board](https://github.com/features/project-management/). We are also looking to use more efficient project management tools to track our documentation and releases and we are open to suggestions for the same.
+
+## How to apply?
+
+The Documentation Working Group is entirely driven by volunteer efforts but we like to keep track of all the contributors who would like to join. It help us set proper workflows, processes and contribution pipelines that allows us to track the work being done and set realistic timelines. You can pitch for several documentation-specific domains that you would like to contribute to.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:57]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+-   **Coordination**: moja global contributors put a lot of focus on documentative approach of communication. Every message written on Slack or the mailing list, if carefully written, can be reused in the future as a resource that can be helpful to new contributors. The coordination efforts relies on finding these resources which can be integrated with the documentation and coordinate the forums and mailing lists.
+
+We are yet to decide on the tools & technologies that we would be employing for our purpose. However some of them that are most commonly employed are [Markdown](https://guides.github.com/features/mastering-markdown/), [Docusaurus](https://docusaurus.io/), [Sphinx](https://www.sphinx-doc.org/), [ReadTheDocs](https://readthedocs.org/), [Moodle](https://moodle.org/), [GitHub Issues](https://guides.github.com/features/issues/), [GitHub Project Board](https://github.com/features/project-management/). We are also looking to use more efficient project management tools to track our documentation and releases and we are open to suggestions for the same.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:53]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+-   **Promotion**: The primary reason why documentation is so much focussed on is because of its potential to develop the community outreach and promotion. As part of the Documentation Working group, we work on promotional content for Social media channels that can be utilized to spread the word about moja global and how people can contribute to our initiative.
+
+-   **Coordination**: moja global contributors put a lot of focus on documentative approach of communication. Every message written on Slack or the mailing list, if carefully written, can be reused in the future as a resource that can be helpful to new contributors. The coordination efforts relies on finding these resources which can be integrated with the documentation and coordinate the forums and mailing lists.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:51]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+-   **Training Material**: Training materials are of utmost need for new developers and contributors at moja global. It helps them upskill at a pace that we like, to make them fit for contributing and plugging in their efforts. moja global hosts its training materials over Moodle learning management system (LMS) which would be expanded over the time.
+
+-   **Website maintenance**:  We love to have people with some experience in Web Development to maintain our [main website](http://moja.global/) and the [community website](https://github.com/moja-global/contributor-website-v2). The skill set depends on the area of work, but the significant part of the website maintenance concerns itself with updating content, writing blogs and content which can be published centrally.
+
+-   **Promotion**: The primary reason why documentation is so much focussed on is because of its potential to develop the community outreach and promotion. As part of the Documentation Working group, we work on promotional content for Social media channels that can be utilized to spread the word about moja global and how people can contribute to our initiative.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:49]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+As part of the Documentation Working group, we work asynchronously with the engineering teams to set a clear Release, feature freeze, and code Freeze dates, which allows us to set our Documentation Freeze and even Release Notes dates.
+
+Remember, [docs or it didn’t happen](https://opensource.com/business/15/8/docs-or-it-didnt-happen).
+
+## What are the different areas we work on?
+
+The Documentation Working Group is not just limited to develop software documentation, release notes and user/developer documentation. We work on a plethora of things which concerns the overall Content Strategy in place for moja global (under development). As an abstraction, some of our work is concerned with the following:
+
+-   **Training Material**: Training materials are of utmost need for new developers and contributors at moja global. It helps them upskill at a pace that we like, to make them fit for contributing and plugging in their efforts. moja global hosts its training materials over Moodle learning management system (LMS) which would be expanded over the time.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:45]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+4.  Merge: The documentation is peer reviewed, and after all suggestions are incorporated, the write-up is merged to the official documentation and hosted.
+
+But where do we plan for documentation releases and developing new documentation pieces? The Documentation issues are created on the respective repositories through GitHub Issues, where they are tracked and accounted for by the stakeholders. All the documentation-related issues would be labelled as “documentation”. At the end of the day, as a volunteer for the Documentation Working group we would be working on:
+
+-   Creating user-story based end user documentation that makes complex things simple so that even a first time user can quickly start off using our projects.
+
+-   Creating modular content that can be reused/repurposed for different formats and different use cases.
+
+-   Adhering to minimalism practices, logically structure our content, and endeavor to make it as findable as possible.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:33]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+2.  Draft: Based on the prior research done, a draft can be prepared which is then reviewed by the stakeholders before signing it off.
+
+3.  Review: After the release of the final draft, the developers/maintainers review the draft and suggest improvements and provide feedback before it is sent as a Pull Request.
+
+4.  Merge: The documentation is peer reviewed, and after all suggestions are incorporated, the write-up is merged to the official documentation and hosted.
+
+But where do we plan for documentation releases and developing new documentation pieces? The Documentation issues are created on the respective repositories through GitHub Issues, where they are tracked and accounted for by the stakeholders. All the documentation-related issues would be labelled as “documentation”. At the end of the day, as a volunteer for the Documentation Working group we would be working on:
+
+-   Creating user-story based end user documentation that makes complex things simple so that even a first time user can quickly start off using our projects.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:29]: Done!

File: Documentation-Working-Group-Proposal.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:27]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+As the name suggests, the primary goal of the Documentation Working Group is to develop documentation for various projects under the moja global umbrella. However, to better understand how to connect our efforts on documentation, an understanding of how documentation is currently developed at moja global is required.
+
+For every project release, a new Documentation plan is created after the epic list for the release is obtained. The epic list describes the bugs that were fixed, the features that were added, and all the trivial/non-trivial changes that were made that hold importance with respect to users and developers. Based on the epic prioritization and bandwidths, the Documentation Working Group commits to documenting the epic list for the release. The Docs Plan is signed off by all the key stakeholders, which includes contributors, ambassadors, coaches and maintainers.
+
+The general documentation development process is very simple, yet very intuitive, for our use-case. It includes the following steps:
+
+1.  Research: Documentation research process is an intensive one and we depend on the developer inputs while performing the research.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:19]: Done!

File: Documentation-Working-Group-Proposal.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+We are setting up a Documentation Working Group, which is primarily aimed at developing high-quality user and developer documentation. The Documentation Working group is geared towards making sure that all of the important software, tools and projects are well-documented with a standard developer experience defined to help new contributors in getting started with the community.
+
+The Documentation Working Group is specifically used for capability building, bringing onboard new contributors, and helping them to get started, while making sure that the influx of new contributions is not hindered by a lack of good documentation. The next sections of this document describe how the Documentation Working Group is coordinated, and what sort of impact it is creating in the large and diverse community that makes up moja global.
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:9]: Done!

File: Documentation-Working-Group-Proposal.md (hunk 1 above)
Comment (HarshCasper) [reply Documentation-Working-Group-Proposal.md:27]: Done!

**Pull request merged**